docker/docker-compose.prod.yaml

```diff
@@ -17,6 +17,10 @@ services:
 
   celeryworker:
     restart: unless-stopped
+    environment:
+      # Number of tasks a celery worker should run before being recreated
+      # https://docs.celeryproject.org/en/latest/userguide/configuration.html#std:setting-worker_max_tasks_per_child
+      CELERYD_MAX_TASKS_PER_CHILD: 10
```

Comment:
Love this idea, but 10 feels like an arbitrarily low number.  I suppose you're experimenting, but I would expect there's startup costs to each worker ...

Reply:
We've have/had `10` running on *-test/dev for a few months without issue, but I haven't tried any other values.

I'm tempted to keep what we've been testing against (`10`), then update test/dev and compare memory usage (to demo).

@pbugni Thoughts?